vehicle/remote_cmd_converter/src/node.cpp

```diff
@@ -16,39 +16,62 @@
 
 #include "remote_cmd_converter/node.hpp"
 
-RemoteCmdConverter::RemoteCmdConverter() : nh_(""), pnh_("~")
+using std::placeholders::_1;
+
+RemoteCmdConverter::RemoteCmdConverter()
+: Node("remote_cmd_converter"),
+  ros_clock_(RCL_ROS_TIME),
+  updater_(this),
+  accel_map_(get_logger()),
+  brake_map_(get_logger())
 {
-  pub_cmd_ = pnh_.advertise<autoware_control_msgs::ControlCommandStamped>("out/control_cmd", 1);
-  pub_current_cmd_ = pnh_.advertise<autoware_vehicle_msgs::RawControlCommandStamped>(
-    "out/latest_raw_control_cmd", 1);
-
-  sub_velocity_ = pnh_.subscribe("in/twist", 1, &RemoteCmdConverter::onVelocity, this);
-  sub_control_cmd_ =
-    pnh_.subscribe("in/raw_control_cmd", 1, &RemoteCmdConverter::onRemoteCmd, this);
-  sub_shift_cmd_ = pnh_.subscribe("in/shift_cmd", 1, &RemoteCmdConverter::onShiftCmd, this);
-  sub_gate_mode_ = pnh_.subscribe("in/current_gate_mode", 1, &RemoteCmdConverter::onGateMode, this);
-  sub_emergency_ = pnh_.subscribe("in/emergency", 1, &RemoteCmdConverter::onEmergency, this);
+  pub_cmd_ = create_publisher<autoware_control_msgs::msg::ControlCommandStamped>(
+    "out/control_cmd", rclcpp::QoS{1});
+  pub_current_cmd_ = create_publisher<autoware_vehicle_msgs::msg::RawControlCommandStamped>(
+    "out/latest_raw_control_cmd", rclcpp::QoS{1});
+
+  sub_velocity_ = create_subscription<geometry_msgs::msg::TwistStamped>(
+    "in/twist", 1, std::bind(&RemoteCmdConverter::onVelocity, this, _1));
+  sub_control_cmd_ = create_subscription<autoware_vehicle_msgs::msg::RawControlCommandStamped>(
+    "in/raw_control_cmd", 1, std::bind(&RemoteCmdConverter::onRemoteCmd, this, _1));
+  sub_shift_cmd_ = create_subscription<autoware_vehicle_msgs::msg::ShiftStamped>(
+    "in/shift_cmd", 1, std::bind(&RemoteCmdConverter::onShiftCmd, this, _1));
+  sub_gate_mode_ = create_subscription<autoware_control_msgs::msg::GateMode>(
+    "in/current_gate_mode", 1, std::bind(&RemoteCmdConverter::onGateMode, this, _1));
+  sub_emergency_ = create_subscription<autoware_control_msgs::msg::EmergencyMode>(
+    "in/emergency", 1, std::bind(&RemoteCmdConverter::onEmergency, this, _1));
 
   // Parameter
-  pnh_.param<double>("ref_vel_gain", ref_vel_gain_, 3.0);
+  ref_vel_gain_ = declare_parameter("ref_vel_gain", 3.0);
 
   // Parameter for Hz check
-  pnh_.param<double>("time_threshold", time_threshold_, 3.0);
+  time_threshold_ = declare_parameter("time_threshold", 3.0);
   double timer_rate;
-  pnh_.param<double>("timer_rate", timer_rate, 10.0);
-  rate_check_timer_ = pnh_.createTimer(ros::Rate(timer_rate), &RemoteCmdConverter::onTimer, this);
+  timer_rate = declare_parameter("timer_rate", 10.0);
```

Comment:
Nitpick: This can be combined with the previous line

Reply:
Done.